internal/service/privatelinkendpointserviceserverless/resource_privatelink_endpoint_service_serverless.go

```diff
@@ -28,6 +30,7 @@ func Resource() *schema.Resource {
 		CreateWithoutTimeout: resourceCreate,
 		ReadWithoutTimeout:   resourceRead,
 		DeleteWithoutTimeout: resourceDelete,
+		UpdateWithoutTimeout: resourceUpdate,
```

Comment:
very weird there wasn't an update here

Reply:
yes, possibly because apart from the comment attribute no other values can be updated

Reply:
i think it is because all attributes had ForceNew

Reply:
yeah so because there was no Update operation configured, all the attributes had to be set with ForceNew otherwise TF throws schema validation error